### DISCUSSION TITLE
Cant use Helper date on request REST API

### DISCUSSION THREAD

**Discussion by marcelo-assain** [Budibase/budibase#9805] [Help] [answered]
Hi Guys, i have a problem. i cant bind date now on bindings of REST API,  i put {{ now "YYYY-MM-d HH:mm:ss" }} is equal 2023-02-24T18:11:55.000Z, but i need "2023-02-24T18:11:55"
I also need to set tomorrow's date with the same format. YYYY-MM-DDTHH:mm:ss
{{ date }} does not work  =(
Thanks!
![image](https://user-images.githubusercontent.com/102670499/221260349-7685b5dd-80be-42b6-a55a-91e069708678.png)

**Comment by melohagan** [comment, marked as answer]
Hey @marcelo-assain 

You'll need to do that handlebars expression on the front-end and pass it through. Here's an example: https://github.com/Budibase/budibase/discussions/9598#discussioncomment-4972633
